src/compass/s1_geocode_metadata.py

```diff
@@ -107,16 +107,19 @@ def run(cfg, burst, fetch_from_scratch=False):
                 continue
 
             dtype = np.single
+            init_value = np.nan
             # layoverShadowMask is last option, no need to change data type
             # and interpolator afterwards
             if layer_name == 'layover_shadow_mask':
                 geocode_obj.data_interpolator = 'NEAREST'
                 dtype = np.byte
+                init_value = 255
```

Comment:
Is this the value we attribute to non-valid pixels?

Reply:
Yes. The value should be 127 which is used in `nisar.workflows.geocode_insar`. I'll add more comments to explain this.